src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryApiServiceSpec.scala

```diff
@@ -204,6 +205,15 @@ class LibraryApiServiceSpec extends BaseServiceSpec with LibraryApiService {
           }
         }
       }
+      "GET on " + libraryGroupsPath - {
+        "should return the all broad users group" in {
+          new RequestBuilder(HttpMethods.GET)(libraryGroupsPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(libraryRoutes) ~> check {
+            status should equal(OK)
+            val respdata = response.entity.asString
+            assert(respdata.contains("all_broad_users"))
```

Comment:
Instead of hard-coding this, it might be better to read the values from the config. This way, if we add more, the test doesn't have to change. Something like:
```
FireCloudConfig.ElasticSearch.discoverGroupNames.foreach { group => assert(respdata.contains(group)) }
```